modules/ossm-operatorhub-remove.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/removing-ossm.adoc
+
+[id="ossm-operatorhub-remove_{context}"]
+= Removing the Operators from OperatorHub
+
+You must remove the Operators to successfully remove {ProductName}. Once you remove the {ProductName} Operator, you must remove the Elasticsearch Operator, Jaeger Operator, and Kiali Operator.
+
+[id="ossm-remove-operator-servicemesh_{context}"]
+== Removing the {ProductName} Operator
+Follow this procedure to remove the {ProductName} Operator.
+
+.Prerequisites
+
+* The {ProductName} Operator must be installed.
+
+.Procedure
+
+. Log into the {product-title} web console.
+
+. Navigate to *Catalogs* -> *Operator Management*.
+
+. Click the *Project* menu, and then click *openshift-operators*.
+
+. Click the *servicemeshoperator* Options menu {kebab}, and then click *Remove Subscription*.
+
+. Select *Also completely remove the Operator from the selected namespace* and click *Remove*.
+
+[id="ossm-remove-operator-elasticsearch_{context}"]
+== Removing the Elasticsearch Operator
+Follow this procedure to remove the Elasticsearch Operator.
+
+.Prerequisites
+
+* The Elasticsearch Operator must be installed.
+
+.Procedure
+
+. Log into the {product-title} web console.
+
+. Navigate to *Catalogs* -> *Operator Management*.
+
+. Click the *Project* menu, and then click *openshift-operators*.
+
+. Click the *elasticsearch-operator* Options menu {kebab}, and then click *Remove Subscription*.
+
+. Select *Also completely remove the Operator from the selected namespace* and click *Remove*.
+
+
+[id="ossm-remove-operator-jaeger_{context}"]
+== Removing the Jaeger Operator
+Follow this procedure to remove the Jaeger Operator.
+
+.Prerequisites
+
+* The Jaeger Operator must be installed.
+
+.Procedure
+
+. Log into the {product-title} web console.
+
+. Navigate to *Catalogs* -> *Operator Management*.
+
+. Click the *Project* menu, and then click *openshift-operators*.
+
+. Click the *jaegeroperator* Options menu {kebab}, and then click *Remove Subscription*.
+
+. Select *Also completely remove the Operator from the selected namespace* and click *Remove*.
+
+[id="ossm-remove-operator-kiali_{context}"]
+== Removing the Jaeger Operator
+Follow this procedure to remove the Kiali Operator.
+
+.Prerequisites
+
+* The Kiali Operator must be installed.
+
+.Procedure
+
+. Log into the {product-title} web console.
+
+. Navigate to *Catalogs* -> *Operator Management*.
+
+. Click the *Project* menu, and then click *openshift-operators*.
+
+. Click the *kiali-ossm* Options menu {kebab}, and then click *Remove Subscription*.
+
+. Select *Also completely remove the Operator from the selected namespace* and click *Remove*.
+
```

Comment:
Should we add a blurb identifying additional steps required to fully purge the install?  Some resources are created by the operator that are not removed through the above procedure.  See https://issues.jboss.org/browse/MAISTRA-846.

This text should look something like:

The operator creates a number of resources, which are not managed by OLM (OperatorHub) and need to be removed manually after uninstalling the Istio operator.  As cluster-admin execute the following commands (where <operator-namespace> is the namespace within which the Istio operator was installed, typically `openshift-operators`):
```
$    oc delete validatingwebhookconfiguration/<operator-namespace>.servicemesh-resources.maistra.io
$    oc delete -n <operator-namespace> daemonset/istio-node
$    oc delete clusterrole/istio-admin
$    oc get crds -o name | grep '.*\.istio\.io' |xargs -r -n 1 oc delete
$    oc get crds -o name | grep '.*\.maistra\.io' |xargs -r -n 1 oc delete
```

Reply:
I added a procedure for this and it will appear in my next PR update.